15_fifteenth_hw/test_human.py

```diff
@@ -0,0 +1,67 @@
+"""
+Test the Human class in the attached file. I will check the coverage. Write as much TCs as possible
+"""
+import pytest
+
+
+@pytest.mark.smoke
+def test_change_gender(create_female_age_30):
+    human = create_female_age_30
+    new_gender = 'male'
+    human.change_gender(new_gender)
+    assert human.gender == new_gender, f'Gender was not changed to {new_gender}'
+
+
+@pytest.mark.regression
+@pytest.mark.xfail
+def test_not_change_incorrect_gender(create_female_age_30):
+    human = create_female_age_30
+    new_gender = 'ololo'
+    human.change_gender(new_gender)
+    assert human.gender == new_gender, f'Gender was changed to {new_gender}'
```

Comment:
this test is incorrect. assert never execute.
You should use pytest.raises to test Exceptions